projects/cdk/abstract/interactive.ts

```diff
@@ -60,7 +60,7 @@ export abstract class AbstractTuiInteractive {
 
     focusVisible = false;
 
-    constructor() {
+    protected constructor() {
```

Comment:
I remember seeing some issues with this approach, but maybe I'm mistaken. @vladimirpotekhin @MarsiBarsi do you remember anything in that regard?

Reply:
![image](https://user-images.githubusercontent.com/12021443/151800246-f841a7c7-c425-4c5c-b216-af9594dce649.png)

Abstract classes still can't be called directly without inheritance

Reply:
I don't remember